### DISCUSSION TITLE
Comparing result save as html format

### DISCUSSION THREAD

**Discussion by bony666** [WinMerge/winmerge#1701] [Q&A]
When (folder compare report) export as html format, only difference result file is saved as html format. How do I save all results as html format.
![wm](https://user-images.githubusercontent.com/57823681/215935888-a00d3943-8fec-44ca-a2e7-b09edbf7b268.png)

**Comment by sdottaka** [comment]
Unfortunately, it is not currently possible to include files identified as identical in the report.

**Reply by IronMan-Nam** [reply]
is this  function already support?

**Reply by sdottaka** [reply]
Even the current latest version 2.16.42.1 still only creates HTML files for files that have differences.

**Reply by IronMan-Nam** [reply]
hope you add this option. Thanks so much
